core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -253,6 +246,8 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
 
         if (msg instanceof Http2Settings) {
             handleHttp2Settings(ctx, (Http2Settings) msg);
+        } else if (msg instanceof RoutingContext) {
```

Comment:
When can this be a `RoutingContext`?

Reply:
Oops... It was cruft. I tried to pass `RoutingContext` before. 😆